src/oc_erchef/apps/chef_db/src/chef_db.erl

```diff
@@ -285,33 +275,10 @@ fetch_org_id(Context, OrgName) ->
 -spec fetch_org_metadata(#context{}, binary() | ?OSC_ORG_NAME) -> not_found | {binary(), binary()}.
 fetch_org_metadata(_, ?OSC_ORG_NAME) ->
     {?OSC_ORG_ID, ?OSC_REQUESTOR_ID};
-fetch_org_metadata(#context{darklaunch = Darklaunch} = Context, OrgName) ->
-    case chef_db_darklaunch:is_enabled(<<"couchdb_organizations">>, Darklaunch) of
-        true ->
-            cdb_fetch_org_metadata(Context, OrgName);
-        false ->
-            sql_fetch_org_metadata(Context, OrgName)
-    end.
-
-%% For couchdb-based orgs, fetch information from the orgname -> metadata cache
-cdb_fetch_org_metadata(#context{reqid = ReqId,
-                                otto_connection = Server}, OrgName) ->
-    case chef_cache:get(org_metadata, OrgName) of
```

Comment:
Should we also rip out the org_guid cache code altogether?  At this point enabling it would be a no-op.  Alternatively, we could see if there are benefits of changing the cahce to support pgsql and start using it again, might be interesting to see what benefits we reap.